app/code/Magento/Catalog/Test/Unit/Model/ProductRepositoryTest.php

```diff
@@ -291,6 +291,7 @@ protected function setUp()
             ->disableOriginalConstructor()
             ->setMethods([])
             ->getMockForAbstractClass();
+        $storeMock->method('getId')->willReturn('1');
```

Comment:
Replace magic number (1) with `const STUB_STORE_ID` that gives more hint what are you actually putting there - if that number has any specific meaning or it's just store ID stub.

Reply:
done